tests/functional/backend/SaveQuestionAttributesTest.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+namespace ls\tests;
+
+use Facebook\WebDriver\WebDriverBy;
+use Facebook\WebDriver\WebDriverExpectedCondition;
+use Facebook\WebDriver\Remote\LocalFileDetector;
+
+/**
+ * @since 2021-11-09
+ * @group questionattribute
+ */
+class SaveQuestionAttributesTest extends TestBaseClassWeb
+{
+    public static $newUserId = null;
+
+    /**
+     * Setup
+     */
+    public static function setupBeforeClass(): void
+    {
+        parent::setupBeforeClass();
+
+        // Permission to everything.
+        \Yii::app()->session['loginID'] = 1;
+
+        $oUser = self::createUserWithPermissions(
+            [
+                "users_name" => "surveyadmin",
```

Comment:
What happen if user already exist (do this test only) : maybe try with random sring ?

Reply:
This is tested on a balnk environment.

Reply:
> This is tested on a blank environment.

By our CI, but each test can be tested on existing environment (i use it a lot when construct test).

I think it's better, but not mandatory :).

Reply:
@olleharstedt did we construct the test globally or only for our CI ?

Reply:
Hm, globally what? The test can make assumption on the data, like "populated database" or "empty database". I'm currently working on a script to generate random test data.

Reply:
We don't really have a good system for database fixtures right now. So all tests more or less work on incidental data created by previous tests. Not so good, really.